mappings/net/minecraft/world/IWorldListener.mapping

```diff
@@ -1,8 +1,8 @@
 CLASS none/aiy net/minecraft/world/IWorldListener
-	METHOD a onSpawnParticle (IZDDDDDD[I)V
+	METHOD a onParticle (IZDDDDDD[I)V
 		ARG 0 typeId
 		ARG 1 ignoreRange
-	METHOD a onPlaySound (Lnone/aam;Lnone/nk;Lnone/nm;DDDFF)V
+	METHOD a onSound (Lnone/aam;Lnone/nk;Lnone/nm;DDDFF)V
```

Comment:
It felt more appropriate with `onParticle` and `onEntityAdded/Removed`. I was thinking maybe `onSoundPlayed` but that sounded weird, and `onSound` doesn't feel wrong to me. Also it doesn't necessarily "play" a sound - it is called when a sound "happens" in the world, both on client and server, right?